crates/driver/src/tests/setup/mod.rs

```diff
@@ -301,6 +327,11 @@ pub struct Solver {
     /// Whether or not solver is allowed to combine multiple solutions into a
     /// new one.
     merge_solutions: bool,
+    /// Whether or not the solver is a JIT-order surplus capturing address
+    surplus_capturing_jit_order_owner: bool,
+    /// Whether or not the solver will be configured (token allowance) for JIT
+    /// orders
+    configure_for_jit_orders: bool,
```

Comment:
I think all solvers should be able to use JIT orders. If we want to have a test case that doesn't make use of JIT orders we should have a way to do that instead of having to disable this capability altogether for a solver.